apps/imageHotspotCreator/src/components/Validation.tsx

```diff
@@ -0,0 +1,28 @@
+import { EntryCard } from "@contentful/f36-components";
+import React from "react";
+import "./ImageSelector/selectImage.css";
+const ValidationPage = ({missedField,fieldMissing}:any) =>{
+    return(
+        <>
+        {missedField.length > 0 &&
+            <div className="missingFieldContainer">
+              <div className="missingFieldImageContainer">
+                <img src={fieldMissing} alt="Field Missing" style={{ height: "100%" }} />
+              </div>
+              <div>
+                It seems you are missed some fields in this content type which is required to use this custom application.
```

Comment:
Consider writing this a little more clearly. "It seems you have missed some fields..."

Reply:
Sure , will do that.